pkg/oc/cli/admin/prune/auth/bindings.go

```diff
@@ -29,7 +32,13 @@ func reapClusterBindings(removedSubject kapi.ObjectReference, c authclient.Inter
 		if len(retainedSubjects) != len(binding.Subjects) {
 			updatedBinding := binding
 			updatedBinding.Subjects = retainedSubjects
-			if _, err := c.Authorization().ClusterRoleBindings().Update(&updatedBinding); err != nil && !kerrors.IsNotFound(err) {
+			coreSubjects, err := convertObjectReference(retainedSubjects)
+			if err != nil {
+				errors = append(errors, err)
+				continue
+			}
+			updatedBinding.UserNames, updatedBinding.GroupNames = authorizationapi.StringSubjectsFor(binding.Namespace, coreSubjects)
```

Comment:
@juanvallejo fyi, this logic, which was previously hidden inside conversions was causing the error, sigh...